result/isislsp.yaml

```diff
@@ -50,9 +38,14 @@ components:
       properties:
         lsp_id:
           description: >-
-            LSP ID of the LSP, e.g. '6400000000010000'.
+            LSP ID of the LSP, in the format, e.g. '640000000001-00-00'.
+            LSP ID is consist of the System ID of a neighbor, the Pseudonode ID, and the LSP number of the LSPs.
+            The last two bytes respectively represent Pseudonode ID and LSP number.
+            A pseudonode is a logical representation of the LAN which is generated by a Designated Intermediate System (DIS)
+            on a LAN segment. If one LSP is not fit by maximum LSP size then thats sent in another LSP with LSP number
```

Comment:
If one LSP exceeds the maximum LSP size then it is sent in another LSP with the LSP number incremented by one.